controllers/scaledobject_controller.go

```diff
@@ -100,6 +101,7 @@ func initScaleClient(mgr manager.Manager, clientset *discovery.DiscoveryClient)
 	), nil
 }
 
+//Reconcile performs reonciliation on the identified Kubernetes object based on the request information passed, returns the result and an error (if any).
```

Comment:
I'd replace `Kubernetes object` with `ScaledObject resource`

Reply:
> I'd replace `Kubernetes object` with `ScaledObject resource`

I have replaced the words.